ui/src/components/group/AddGroup.js

```diff
@@ -130,6 +137,12 @@ export default class AddGroup extends React.Component {
         if (!name) {
             return;
         }
+        if (name.includes('/')) {
```

Comment:
let's use this https://github.com/yahoo/athenz/blob/master/core/zms/src/main/rdl/Names.tdl#L67 regex pattern and perform a check against it for the member name